homeassistant/components/ps4/media_player.py

```diff
@@ -0,0 +1,375 @@
+        self._source_list = list(sorted(self._games.values()))
+
+    def load_games(self):
+        """Load games for sources."""
+        g_file = self._games_filename
+        try:
+            games = load_json(g_file)
+            return games
+        except FileNotFoundError:
+            games = {}
+            self.save_games(games)
+            self.load_games()
+
+    def save_games(self, games):
+        """Save games to file."""
+        g_file = self._games_filename
+        try:
+            save_json(g_file, games)
+        except OSError as error:
+            _LOGGER.error("Could not save game list, %s", error)
+
+    def add_games(self, title_id, app_name):
+        """Add games to list."""
+        games = self._games
+        if title_id is not None and title_id not in games:
+            game = {title_id: app_name}
+            games.update(game)
+            self.save_games(games)
+
+    def get_device_info(self, status):
+        """Return device info for registry."""
+        _sw_version = status['system-version']
+        _sw_version = _sw_version[1:4]
+        sw_version = _sw_version[0] + '.' + _sw_version[1:]
+        self._info = {
+            'name': status['host-name'],
+            'model': 'PlayStation 4',
+            'identifiers': {
+                (PS4_DOMAIN, status['host-id'])
+            },
+            'manufacturer': 'Sony Interactive Entertainment Inc.',
+            'sw_version': sw_version
+        }
+        self._unique_id = status['host-id']
+
+    @property
+    def device_info(self):
+        """Return information about the device."""
+        return self._info
+
+    @property
+    def unique_id(self):
+        """Return Unique ID for entity."""
+        return self._unique_id
+
+    @property
+    def entity_picture(self):
+        """Return picture."""
+        if self._state == STATE_OFF:
+            return None
+
+        image_hash = self.media_image_hash
+        if image_hash is not None:
+            return ENTITY_IMAGE_URL.format(
+                self.entity_id, self.access_token, image_hash)
+
+        if self._media_content_id is None:
+            return None
+
+    @property
+    def name(self):
+        """Return the name of the device."""
+        return self._name
+
+    @property
+    def state(self):
+        """Return the state of the device."""
+        return self._state
+
+    @property
+    def icon(self):
+        """Icon."""
+        return ICON
+
+    @property
+    def media_content_id(self):
+        """Content ID of current playing media."""
+        return self._media_content_id
+
+    @property
+    def media_content_type(self):
+        """Content type of current playing media."""
+        return MEDIA_TYPE_MUSIC
+
+    @property
+    def media_image_url(self):
+        """Image url of current playing media."""
+        if self._media_content_id is None:
+            return MEDIA_IMAGE_DEFAULT
+        try:
+            return self._media_image
```

Comment:
Is this really a dict?

Reply:
Lol no. Was leftover code from an old version.